src/test/java/teammates/test/cases/webapi/DeleteFeedbackResponseActionTest.java

```diff
@@ -23,13 +42,168 @@ protected String getRequestMethod() {
     @Test
     @Override
     protected void testExecute() throws Exception {
-        // TODO
+        useTypicalDataBundle();
+
+        loginAsStudent(student1InCourse1.getGoogleId());
+
+        ______TS("Not enough parameters");
+
+        verifyHttpParameterFailure();
+        verifyHttpParameterFailure(Const.ParamsNames.INTENT, Intent.STUDENT_SUBMISSION.toString());
+
+        ______TS("Typical success case, student");
+
+        String[] params = new String[] {
+                Const.ParamsNames.INTENT, Intent.STUDENT_SUBMISSION.toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, typicalResponse.getId(),
+        };
+
+        DeleteFeedbackResponseAction a = getAction(params);
+        JsonResult r = getJsonResult(a);
+
+        assertEquals(HttpStatus.SC_OK, r.getStatusCode());
+
+        // response is deleted
+        assertNull(logic.getFeedbackResponse(typicalResponse.getId()));
+
+        ______TS("Typical success case, instructor");
+
+        loginAsInstructor(instructor1OfCourse1.getGoogleId());
+
+        params = new String[] {
+                Const.ParamsNames.INTENT, Intent.INSTRUCTOR_SUBMISSION.toString(),
+                Const.ParamsNames.FEEDBACK_RESPONSE_ID, typicalResponse3.getId(),
+        };
+
+        a = getAction(params);
+        r = getJsonResult(a);
+
+        assertEquals(HttpStatus.SC_OK, r.getStatusCode());
+
+        // response is deleted
+        assertNull(logic.getFeedbackResponse(typicalResponse3.getId()));
     }
 
     @Test
     @Override
     protected void testAccessControl() throws Exception {
```

Comment:
Similar to #9725, shall we test for 
```
        verifyInstructorCanSeeQuestionIfInModeration(feedbackQuestion);
        verifySessionOpenExceptForModeration(feedbackSession);
        verifyNotPreview();
```

@xpdavid

Reply:
Please add test case here  similar to #9725,
preview mode, session not open etc..

Reply:
How about `verifyInstructorCanSeeQuestionIfInModeration(feedbackQuestion);`